code/datums/position_point_vector.dm

```diff
@@ -0,0 +1,195 @@
+//Designed for things that need precision trajectories like projectiles.
+//Don't use this for anything that you don't absolutely have to use this with (like projectiles!) because it isn't worth using a datum unless you need accuracy down to decimal places in pixels.
+
+/proc/return_precise_position(atom/A)

```

Comment:
You claim this is a precise position, yet you don't include matrices offsets.
0/10 go back to school mehki,

Reply:
>months later
>still not done
>procs literally already exist to get x/y offsets from matrices

Reply:
;_;

Reply:
also make these into macros

Reply:
it's precise enough